<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>将棋盤Q</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>ファイル(F)</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="actionOpenKifuFile"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionSaveBoardImage"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="Edit">
    <property name="title">
     <string>編集(E)</string>
    </property>
    <widget class="QMenu" name="copyGameRecord">
     <property name="title">
      <string>棋譜コピー</string>
     </property>
     <addaction name="actionCopyKIF"/>
     <addaction name="actionCopyKI2"/>
     <addaction name="actionCopyCSA"/>
     <addaction name="actionCopyUSICurrent"/>
     <addaction name="actionCopyUSIAll"/>
     <addaction name="actionCopyJKF"/>
     <addaction name="actionCopyUSEN"/>
    </widget>
    <widget class="QMenu" name="shogiPositionCopy">
     <property name="title">
      <string>局面コピー</string>
     </property>
     <addaction name="actionCopySFEN"/>
     <addaction name="actionCopyBOD"/>
    </widget>
    <addaction name="copyGameRecord"/>
    <addaction name="shogiPositionCopy"/>
    <addaction name="actionPasteKifu"/>
    <addaction name="actionSfenCollectionViewer"/>
    <addaction name="separator"/>
    <addaction name="actionCopyBoardToClipboard"/>
    <addaction name="separator"/>
    <addaction name="actionStartEditPosition"/>
    <addaction name="actionEndEditPosition"/>
    <addaction name="actionSetHiratePosition"/>
    <addaction name="actionSetTsumePosition"/>
    <addaction name="actionReturnAllPiecesToStand"/>
    <addaction name="separator"/>
    <addaction name="actionSwapSides"/>
    <addaction name="actionChangeTurn"/>
   </widget>
   <widget class="QMenu" name="Display">
    <property name="title">
     <string>表示(I)</string>
    </property>
    <addaction name="actionFlipBoard"/>
    <addaction name="actionEnlargeBoard"/>
    <addaction name="actionShrinkBoard"/>
    <addaction name="separator"/>
    <addaction name="actionToolBar"/>
   </widget>
   <widget class="QMenu" name="Game">
    <property name="title">
     <string>対局(G)</string>
    </property>
    <addaction name="actionStartGame"/>
    <addaction name="actionCSA"/>
    <addaction name="separator"/>
    <addaction name="actionResign"/>
    <addaction name="actionUndoMove"/>
    <addaction name="actionMakeImmediateMove"/>
    <addaction name="actionBreakOffGame"/>
    <addaction name="separator"/>
    <addaction name="actionAnalyzeKifu"/>
    <addaction name="actionCancelAnalyzeKifu"/>
    <addaction name="separator"/>
    <addaction name="actionTsumeShogiSearch"/>
    <addaction name="actionStopTsumeSearch"/>
    <addaction name="separator"/>
    <addaction name="actionJishogiScore"/>
    <addaction name="actionNyugyokuDeclaration"/>
   </widget>
   <widget class="QMenu" name="Setting">
    <property name="title">
     <string>設定(S)</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>言語設定</string>
     </property>
     <addaction name="actionLanguageSystem"/>
     <addaction name="actionLanguageJapanese"/>
     <addaction name="actionLanguageEnglish"/>
    </widget>
    <addaction name="actionEngineSettings"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>ヘルプ(H)</string>
    </property>
    <addaction name="actionOpenWebsite"/>
    <addaction name="actionUsage"/>
    <addaction name="separator"/>
    <addaction name="actionVersionInfo"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Edit"/>
   <addaction name="Display"/>
   <addaction name="Game"/>
   <addaction name="Setting"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenKifuFile"/>
   <addaction name="actionSave"/>
   <addaction name="actionStartGame"/>
   <addaction name="actionResign"/>
   <addaction name="actionUndoMove"/>
   <addaction name="actionMakeImmediateMove"/>
   <addaction name="actionBreakOffGame"/>
   <addaction name="actionAnalyzeKifu"/>
   <addaction name="actionTsumeShogiSearch"/>
   <addaction name="actionFlipBoard"/>
   <addaction name="actionEnlargeBoard"/>
   <addaction name="actionShrinkBoard"/>
   <addaction name="actionCSA"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionSave.svg</normaloff>:/images/actions/actionSave.svg</iconset>
   </property>
   <property name="text">
    <string>上書き保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionSaveAs.svg</normaloff>:/images/actions/actionSaveAs.svg</iconset>
   </property>
   <property name="text">
    <string>名前を付けて保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSaveBoardImage">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionSaveBoardImage.svg</normaloff>:/images/actions/actionSaveBoardImage.svg</iconset>
   </property>
   <property name="text">
    <string>将棋盤の画像をファイルに保存</string>
   </property>
  </action>
  <action name="actionPasteKifu">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionPasteKifu.svg</normaloff>:/images/actions/actionPasteKifu.svg</iconset>
   </property>
   <property name="text">
    <string>棋譜貼り付け</string>
   </property>
  </action>
  <action name="actionCopyBoardToClipboard">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCopyBoardToClipboard.svg</normaloff>:/images/actions/actionCopyBoardToClipboard.svg</iconset>
   </property>
   <property name="text">
    <string>将棋盤画像コピー</string>
   </property>
  </action>
  <action name="actionFlipBoard">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionFlipBoard.svg</normaloff>:/images/actions/actionFlipBoard.svg</iconset>
   </property>
   <property name="text">
    <string>盤面の回転</string>
   </property>
  </action>
  <action name="actionStartGame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionStartGame.svg</normaloff>:/images/actions/actionStartGame.svg</iconset>
   </property>
   <property name="text">
    <string>対局</string>
   </property>
  </action>
  <action name="actionResign">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionResign.svg</normaloff>:/images/actions/actionResign.svg</iconset>
   </property>
   <property name="text">
    <string>投了</string>
   </property>
  </action>
  <action name="actionBreakOffGame">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionBreakOffGame.svg</normaloff>:/images/actions/actionBreakOffGame.svg</iconset>
   </property>
   <property name="text">
    <string>中断</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionQuit.svg</normaloff>:/images/actions/actionQuit.svg</iconset>
   </property>
   <property name="text">
    <string>終了</string>
   </property>
  </action>
  <action name="actionAnalyzeKifu">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionAnalyzeKifu.svg</normaloff>:/images/actions/actionAnalyzeKifu.svg</iconset>
   </property>
   <property name="text">
    <string>棋譜解析</string>
   </property>
  </action>
  <action name="actionStartEditPosition">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionStartEditPosition.svg</normaloff>:/images/actions/actionStartEditPosition.svg</iconset>
   </property>
   <property name="text">
    <string>局面編集開始</string>
   </property>
  </action>
  <action name="actionSetHiratePosition">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionSetHiratePosition.svg</normaloff>:/images/actions/actionSetHiratePosition.svg</iconset>
   </property>
   <property name="text">
    <string>平手初期配置</string>
   </property>
  </action>
  <action name="actionSetTsumePosition">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionSetTsumePosition.svg</normaloff>:/images/actions/actionSetTsumePosition.svg</iconset>
   </property>
   <property name="text">
    <string>詰将棋初期配置</string>
   </property>
  </action>
  <action name="actionReturnAllPiecesToStand">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionReturnAllPiecesToStand.svg</normaloff>:/images/actions/actionReturnAllPiecesToStand.svg</iconset>
   </property>
   <property name="text">
    <string>全ての駒を駒台へ</string>
   </property>
  </action>
  <action name="actionSwapSides">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionSwapSides.svg</normaloff>:/images/actions/actionSwapSides.svg</iconset>
   </property>
   <property name="text">
    <string>先後反転</string>
   </property>
  </action>
  <action name="actionChangeTurn">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionChangeTurn.svg</normaloff>:/images/actions/actionChangeTurn.svg</iconset>
   </property>
   <property name="text">
    <string>手番変更</string>
   </property>
  </action>
  <action name="actionEngineSettings">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionEngineSettings.svg</normaloff>:/images/actions/actionEngineSettings.svg</iconset>
   </property>
   <property name="text">
    <string>エンジン設定</string>
   </property>
  </action>
  <action name="actionOpenWebsite">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionOpenWebsite.svg</normaloff>:/images/actions/actionOpenWebsite.svg</iconset>
   </property>
   <property name="text">
    <string>ホームページ</string>
   </property>
  </action>
  <action name="actionUsage">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionUsage.svg</normaloff>:/images/actions/actionUsage.svg</iconset>
   </property>
   <property name="text">
    <string>使い方</string>
   </property>
  </action>
  <action name="actionVersionInfo">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionVersionInfo.svg</normaloff>:/images/actions/actionVersionInfo.svg</iconset>
   </property>
   <property name="text">
    <string>バージョン情報</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionAboutQt.svg</normaloff>:/images/actions/actionAboutQt.svg</iconset>
   </property>
   <property name="text">
    <string>Qtについて</string>
   </property>
  </action>
  <action name="actionCopyKIF">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCopyKIF.svg</normaloff>:/images/actions/actionCopyKIF.svg</iconset>
   </property>
   <property name="text">
    <string>KIF形式</string>
   </property>
  </action>
  <action name="actionCopyKI2">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCopyKI2.svg</normaloff>:/images/actions/actionCopyKI2.svg</iconset>
   </property>
   <property name="text">
    <string>KI2形式</string>
   </property>
  </action>
  <action name="actionCopyCSA">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCopyCSA.svg</normaloff>:/images/actions/actionCopyCSA.svg</iconset>
   </property>
   <property name="text">
    <string>CSA形式</string>
   </property>
  </action>
  <action name="actionCopyUSICurrent">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCopyUSICurrent.svg</normaloff>:/images/actions/actionCopyUSICurrent.svg</iconset>
   </property>
   <property name="text">
    <string>USI形式（現在の指し手まで）</string>
   </property>
  </action>
  <action name="actionCopySFEN">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCopySFEN.svg</normaloff>:/images/actions/actionCopySFEN.svg</iconset>
   </property>
   <property name="text">
    <string>SFEN形式</string>
   </property>
  </action>
  <action name="actionCopyBOD">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCopyBOD.svg</normaloff>:/images/actions/actionCopyBOD.svg</iconset>
   </property>
   <property name="text">
    <string>BOD形式</string>
   </property>
  </action>
  <action name="actionNewGame">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionNewGame.svg</normaloff>:/images/actions/actionNewGame.svg</iconset>
   </property>
   <property name="text">
    <string>新規</string>
   </property>
  </action>
  <action name="actionOpenKifuFile">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionOpenKifuFile.svg</normaloff>:/images/actions/actionOpenKifuFile.svg</iconset>
   </property>
   <property name="text">
    <string>開く</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUndoMove">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionUndoMove.svg</normaloff>:/images/actions/actionUndoMove.svg</iconset>
   </property>
   <property name="text">
    <string>待った</string>
   </property>
  </action>
  <action name="actionMakeImmediateMove">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionMakeImmediateMove.svg</normaloff>:/images/actions/actionMakeImmediateMove.svg</iconset>
   </property>
   <property name="text">
    <string>すぐ指させる</string>
   </property>
  </action>
  <action name="actionEnlargeBoard">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionEnlargeBoard.svg</normaloff>:/images/actions/actionEnlargeBoard.svg</iconset>
   </property>
   <property name="text">
    <string>将棋盤の拡大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionShrinkBoard">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionShrinkBoard.svg</normaloff>:/images/actions/actionShrinkBoard.svg</iconset>
   </property>
   <property name="text">
    <string>将棋盤の縮小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionEndEditPosition">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionEndEditPosition.svg</normaloff>:/images/actions/actionEndEditPosition.svg</iconset>
   </property>
   <property name="text">
    <string>局面編集終了</string>
   </property>
  </action>
  <action name="actionTsumeShogiSearch">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionTsumeShogiSearch.svg</normaloff>:/images/actions/actionTsumeShogiSearch.svg</iconset>
   </property>
   <property name="text">
    <string>詰み探索</string>
   </property>
  </action>
  <action name="actionStopTsumeSearch">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCancelTsumeShogiSearch.svg</normaloff>:/images/actions/actionCancelTsumeShogiSearch.svg</iconset>
   </property>
   <property name="text">
    <string>詰み探索中止</string>
   </property>
  </action>
  <action name="actionJishogiScore">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionJishogiScore.svg</normaloff>:/images/actions/actionJishogiScore.svg</iconset>
   </property>
   <property name="text">
    <string>持将棋点数</string>
   </property>
  </action>
  <action name="actionNyugyokuDeclaration">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionNyugyokuDeclaration.svg</normaloff>:/images/actions/actionNyugyokuDeclaration.svg</iconset>
   </property>
   <property name="text">
    <string>入玉宣言</string>
   </property>
  </action>
  <action name="actionCopyUSIAll">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCopyUSIAll.svg</normaloff>:/images/actions/actionCopyUSIAll.svg</iconset>
   </property>
   <property name="text">
    <string>USI形式（全て）</string>
   </property>
  </action>
  <action name="actionCopyJKF">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCopyJKF.svg</normaloff>:/images/actions/actionCopyJKF.svg</iconset>
   </property>
   <property name="text">
    <string>JSON棋譜フォーマット</string>
   </property>
  </action>
  <action name="actionCopyUSEN">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCopyUSIAll.svg</normaloff>:/images/actions/actionCopyUSIAll.svg</iconset>
   </property>
   <property name="text">
    <string>USEN</string>
   </property>
  </action>
  <action name="actionCSA">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCSA.svg</normaloff>:/images/actions/actionCSA.svg</iconset>
   </property>
   <property name="text">
    <string>通信対局（CSA）</string>
   </property>
  </action>
  <action name="actionCancelAnalyzeKifu">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionCancelAnalyzeKifu.svg</normaloff>:/images/actions/actionCancelAnalyzeKifu.svg</iconset>
   </property>
   <property name="text">
    <string>棋譜解析中止</string>
   </property>
  </action>
  <action name="actionMenuWindow">
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionMenuWindow.svg</normaloff>:/images/actions/actionMenuWindow.svg</iconset>
   </property>
   <property name="text">
    <string>メニュー</string>
   </property>
  </action>
  <action name="actionLanguageSystem">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionLanguageSystem.svg</normaloff>:/images/actions/actionLanguageSystem.svg</iconset>
   </property>
   <property name="text">
    <string>システム設定に従う</string>
   </property>
  </action>
  <action name="actionLanguageJapanese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionLanguageJapanese.svg</normaloff>:/images/actions/actionLanguageJapanese.svg</iconset>
   </property>
   <property name="text">
    <string>日本語</string>
   </property>
  </action>
  <action name="actionLanguageEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionLanguageEnglish.svg</normaloff>:/images/actions/actionLanguageEnglish.svg</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/shogiboardq.qrc">
     <normaloff>:/images/actions/actionToolbar.svg</normaloff>:/images/actions/actionToolbar.svg</iconset>
   </property>
   <property name="text">
    <string>ツールバー</string>
   </property>
  </action>
  <action name="actionSfenCollectionViewer">
   <property name="text">
    <string>局面集ビューア</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/shogiboardq.qrc"/>
 </resources>
 <connections/>
</ui>
